src/app/Containers/PreviewContainer/index.unit.test.js

```diff
@@ -0,0 +1,60 @@
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import { Provider } from 'react-redux';
+import { mount } from 'enzyme';
+
+import { SUCCESS, ERROR, LOADING } from '../../../common/constants';
+import PreviewContainer from '.';
+import { Error, Spinner } from '../../../common/Components';
+
+const middlewares = [thunk];
+const mockStore = configureStore(middlewares);
+
+const getContainerWithSpecificState = (state) => {
+  const store = mockStore(state);
+  let container = mount(
+    <Provider store={store}>
+      <PreviewContainer />
+    </Provider>,
+  );
+
+  return container;
+};
+
+describe('PreviewContainer', () => {
+  it('Renders the Spinner component when the state value is LOADING', () => {
+    const state = { popularMovies: { state: LOADING, content: null } };
+
+    const container = getContainerWithSpecificState(state);
+
+    expect(container.find(Spinner)).toHaveLength(1);
+  });
+
+  it('Renders the Error component when the state value is ERROR', () => {
```

Comment:
Let's try to keep consistency on the tests capitalization. Some tests start with lower case, and some others don't